### PR TITLE
ADO-13538 Notification centre restyle

### PR DESCRIPTION
**Pull request by ryan-reddekopp**
restyled the notification centre to look like the new marketing mockups
-added a drop-shadow variable to the variables doc

Screen Shot 2020-07-20 at 11 17 34 AM

Message Modal

🎨

ADO LINK: https://dev.azure.com/AMA-Ent/AMA-Ent/_workitems/edit/13538

### PR Review Checklist
To be completed by the person who opens the PR. Use strikethroughs for items that are not applicable in list.
- [ ] I have reviewed my own PR to check syntax & logic, removed unnecessary/old code, made sure everything aligns with our style guide and BEM.
- [ ] I've added new components to the style guide (or have a PR in to add them).
- [ ] Any applicable version numbers have been updated.
- [ ] I've tested on mobile, tablet, and desktop, as well as across all of the browsers we support.
- [ ] I've proof-read all text for legibility, proper grammar, punctuation, and capitalization (titlecase).
- [ ] I have written a detailed PR message explaining what is being changed and why
- [ ] I’ve linked to any relevant VSTS tickets
- [ ] I’ve added the appropriate review symbols to my PR - (elephant) for dev review, (art) for design

### CODE REVIEW

File: app/assets/stylesheets/v3/layout/variables.scss [hunk 1]
```diff
@@ -107,6 +107,8 @@ $base-padding: 1rem;
 $base-border: 1px solid $stone;
 // Radius
 $base-radius: 9px;
+// Drop-shadow
+$base-box-shadow: 2.5px 4.33px 6.86px 0.14px rgba(24, 24, 24, 0.35)!important;
```
Comment (sineade) [inline app/assets/stylesheets/v3/layout/variables.scss:111]: does this need to be !important? I had added this variable on my branch and I didn't need an !important on it.

Comment (ryan-reddekopp) [reply app/assets/stylesheets/v3/layout/variables.scss:111]: It does! There is another class specifically for this div's transition that says not to apply a drop shadow.
```
.off-canvas.is-transition-overlap.is-open {
    box-shadow: none;
}
```
I believe this is a foundation class that I needed to override.

Comment (ryan-reddekopp) [reply app/assets/stylesheets/v3/layout/variables.scss:111]: It does! There is another class specifically for this div's transition that takes priority and says not to apply a drop shadow.
```
.off-canvas.is-transition-overlap.is-open {
    box-shadow: none;
}```
I believe this is a foundation class that I needed to override.

Comment (sineade) [reply app/assets/stylesheets/v3/layout/variables.scss:111]: okay! 👍

**Pull request merged**